app/src/main/java/com/codesoom/assignment/product/application/ProductCommand.java

```diff
@@ -0,0 +1,61 @@
+package com.codesoom.assignment.product.application;
+
+import com.codesoom.assignment.product.domain.Product;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+@lombok.Generated
+public class ProductCommand {
+
+    @lombok.Generated
+    @Getter
+    @Builder
+    @ToString
+    public static class Register {
+        private final String name;
+
+        private final String maker;
+
+        private final Long price;
+
+        private final String imageUrl;
+
+        public Product toEntity() {
+            return Product.builder()
+                    .name(name)
+                    .maker(maker)
+                    .price(price)
+                    .imageUrl(imageUrl)
+                    .build();
+        }
+
+    }
+
+    @lombok.Generated
+    @Getter
+    @Builder
+    @ToString
+    public static class UpdateReq {
```

Comment:
이름을 지을 때 가급적이면 생략하지 않는 습관을 만들어 보도록 합시다.

```suggestion
    public static class UpdateRequest {
```